packages/astro/src/assets/fonts/config.ts

```diff
@@ -1,36 +1,95 @@
 import { z } from 'zod';
 import { LOCAL_PROVIDER_NAME } from './constants.js';
 
-// TODO: jsdoc for everything, most of those end up in the public AstroConfig type
-
 const weightSchema = z.union([z.string(), z.number()]);
 const styleSchema = z.enum(['normal', 'italic', 'oblique']);
 
 const familyPropertiesSchema = z.object({
+	/**
+	 * A [font weight](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight). If the associated font is a [variable font](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_fonts/Variable_fonts_guide), you can specify a range of weights:
+	 *
+	 * ```js
+	 * weight: "100 900"
+	 * ```
+	 */
 	weight: weightSchema,
+	/**
+	 * A [font style](https://developer.mozilla.org/en-US/docs/Web/CSS/font-style).
+	 */
 	style: styleSchema,
+	/**
+	 * @default `"swap"`
+	 * 
+	 * A [font display](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display).
+	 */
 	display: z.enum(['auto', 'block', 'swap', 'fallback', 'optional']).optional(),
+	/**
+	 * A [unicode range](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/unicode-range).
+	 */
 	unicodeRange: z.array(z.string()).nonempty().optional(),
+	/**
+	 * A [font stretch](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-stretch).
+	 */
 	stretch: z.string().optional(),
+	/**
+	 * Font [feature settings](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-feature-settings).
+	 */
 	featureSettings: z.string().optional(),
+	/**
+	 * Font [variation settings](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-variation-settings).
+	 */
 	variationSettings: z.string().optional(),
 });
 
 const fallbacksSchema = z.object({
+	/**
+	 * @default `["sans-serif"]`
+	 *
+	 * An array of fonts to use when your chosen font is unavailable, or loading. Fallbacks are applied from left to right:
+	 *
+	 * ```js
+	 * fallbacks: ["CustomFont", "serif"]
+	 * ```
+	 *
+	 * If you don't want any fallbacks, use an empty array:
+	 *
+	 * ```js
+	 * fallbacks: []
+	 * ```
+	 *
+	 * If `optimizedFallbacks` is set to `true`, we look for the last fallback and check if it's a [generic family name](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#generic-name). If so, we try to generate an [optimized fallback](https://developer.chrome.com/blog/font-fallbacks) using font metrics.
```

Comment:
Maybe this belongs in the section about optimizedFallBacks, since it's only valid when its true?

Reply:
In the docs, it's in a note and i think it makes sense to explain it here because it explains how fallbacks work, not how optimizedFallbacks work

Reply:
This doesn't bother me but perhaps what is confusing for Sarah is the wording. Does the following would be a better fit for this section:
```suggestion
	 * By default and when `optimizedFallbacks` is not disabled, we look for the last fallback and check if it's a [generic family name](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#generic-name). If so, we try to generate an [optimized fallback](https://developer.chrome.com/blog/font-fallbacks) using font metrics.
```
Or something like that... Because I think it makes more sense here. However, the user might not know yet that `true` is the default for `optimizedFallbacks`, so we might need to be more explicit.

Reply:
Would this be true?? I think we can simplify wording a lot here by just saying what happens. (I think we can even avoid "by default" in the wording below):

> * If the last font in the `fallbacks` array is a [generic family name](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#generic-name),  an [optimized fallback](https://developer.chrome.com/blog/font-fallbacks) using font metrics will be generated. To disable this optimization, set `optimizedFallbacks` to false.

Reply:
Well, that's what I understood from T&D (although not active 😅 ). So, if Florian confirms, yeah I think your suggestion would be both correct and better! 🙌🏽

Reply:
```suggestion

	 * If the last font in the `fallbacks` array is a [generic family name](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#generic-name), an [optimized fallback](https://developer.chrome.com/blog/font-fallbacks) using font metrics will be generated. To disable this optimization, set `optimizedFallbacks` to false.
```